### PR TITLE
Update events.md

### PR DESCRIPTION
**Pull request by dario-ds**
I was playing with this for some time and I think this should make it a lot clearer. Might have got some functionality wrong, but it didn't seem like I had to create a project page for it to actually work :)

### CODE REVIEW

File: docs/introduction/events.md [hunk 1]
```diff
@@ -1,26 +1,29 @@
 # Events
 
-Events fall into two categories:
-- journal: tasks in the journal
-- project: tasks in other notes
+Events are any blocks in your database that have both a time dimension and a Logseq task property associated with them (e.g. TODO, DOING, LATER, NOW, WAITING)
```
Comment (jpjpjp) [inline docs/introduction/events.md:3]: Or any block that is a child of the [[Daily Log]] block.  (I think)

File: docs/introduction/events.md [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
+Events can be in your journal or in any other page. Note that there are slight differences in setting up events in the journal vs in other pages. 
 
-Use the date of the diary as the date of the task, and when the task starts with a time, treat it as the specific time of the task.
+Also note that if you want events to show up in your project dashboard, you have have to insert those blocks in the relevant project page.
 
-Example:
-![journal tasks](../../screenshots/journal-tasks.png)
+## Journal page only
+
+Enter the task on the journal page for that date. If the task starts at a certain time, specify the time at the start of the block. You can also specify the end time, or alternatively the plugin will use your default time for events when displaying the event in your calendar.
```
Comment (jpjpjp) [inline docs/introduction/events.md:11]: Is this the right place to introduce the idea of using the #milestone tag as well?   I'll confess that I don't quite get how a "milestone" is different than an "event", but based on this video https://www.youtube.com/watch?v=m2RFQHmA-zA this seems to be another way of creating events on the journal page.

File: docs/introduction/events.md [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
 
+### Example:
+![journal tasks](../../screenshots/journal-tasks.png)
 
-## Project
+## All pages (including non-project pages)
```
[the remark this answers is not in the document]
Comment (dario-ds) [reply docs/introduction/events.md:16]: I agree, that is better. I'm not sure how best to use GitHub (this is the first time I'm using it collaboratively)


All non-journal pages (including pages that are not specified as 'Projects' in the plugin)

File: docs/introduction/events.md (hunk 1 above)
Comment (haydenull) [reply docs/introduction/events.md:3]: Blocks under the Daily Log are mainly used for interstitial journals. starting with version 2.5.0, they will be displayed separately on the new Daily Log page.

File: docs/introduction/events.md (hunk 2 above)
Comment (haydenull) [reply docs/introduction/events.md:11]: I will add a separate note about milestone later.

**Review by haydenull — APPROVED** [review]
LGTM

**Pull request merged**